common/src/java/org/apache/hadoop/hive/common/type/Timestamp.java

```diff
@@ -88,6 +88,10 @@ public Timestamp(Timestamp t) {
     this(t.localDateTime);
   }
 
+  public Timestamp(java.sql.Timestamp ts) {
```

Comment:
Could you remove this new constructor and use existing methods in `Timestamp` class to create the object (maybe you can encapsulate logic in utility method in JdbcSerDe)? `java.sql.Timestamp` is used in some places in Hive with UTC time zone vs system time zone and this constructor may make things confusing.

Reply:
Make Sense. Done